packages/vuetify/src/components/VSelect/VSelectList.ts

```diff
@@ -164,7 +165,7 @@ export default mixins(Colorable, Themeable).extend({
           // Default behavior in list does not
           // contain aria-selected by default
           'aria-selected': String(value),
-          'aria-labelledby': this.genLabelledBy(item),
+          'aria-labelledby': `${this.genLabelledBy(item)}-${index}`,
```

Comment:
Since we're using now `index` I guess the `const text = escapeHTML(this.getText(item).split(' ').join('-').toLowerCase())` part in `genLabelledBy` is not necessary anymore

Reply:
I don't think so. In one scenario with multiple selects the Id's can be duplicated. This part doesn't avoid all possible duplications, but it's not common to have multiple selects with same options.

In case I'm wrong, please let me know.

Reply:
This should be handled by `_uid`
```ts
genLabelledBy (item: object) {
      const text = escapeHTML(this.getText(item).split(' ').join('-').toLowerCase())

      return `${text}-list-item-${this._uid}`
    },
```
Can you check it?
Also it's very common to have multiple selets with same options, for example you can have selects with nully value ('not applicable', 'other', etc)

Reply:
I will recheck the use of `_uid` in this function.

Reply:
@jacekkarczmarczyk I checked again your point and isn't necessary anymore the text. Only `list-item-${this._uid}` with the index. I didn't get your point.
Thanks for your review.